### PR TITLE
Upload release files to CDN as part of deploy script

### PR DESCRIPTION
**Pull request by jonsnyder**
## Description

Create a directory named the same as the version in CDN, and upload alloy.js, and alloy.min.js there.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

**Review by jfkhoury — APPROVED** [review]

File: scripts/sftpCommands.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+VERSION=$1
+
+# When sftp commands are run in batch mode (-b flag), commands prefixed with "-" do not fail the
+# entire script. By prefixing mkdir with "-", this will not fail if the directory already exists.
+echo "-mkdir $VERSION"
+echo "cd $VERSION"
+echo "put ./dist/alloy.js"
+echo "put ./dist/alloy.min.js"
+echo "bye"
```
Comment (jfkhoury) [inline scripts/sftpCommands.sh:11]: Can we verify that it uploaded or it's overkill?

**Review by Aaronius — APPROVED** [review]
🎆

File: scripts/sftpCommands.sh (hunk 1 above)
Comment (jonsnyder) [reply scripts/sftpCommands.sh:11]: Good idea. I've added some commands to this script to check that the files are available on the CDN.

File: scripts/deploy.sh [hunk 2]
```diff
@@ -40,3 +40,22 @@ npm run build
...
+# verify the files are available on the cdn
+ALLOY_MIN_JS="https://cdn1.adoberesources.net/alloy/${VERSION}/alloy.min.js"
+STATUS_CODE=$(curl -o /dev/null -Isw '%{http_code}\n' $ALLOY_MIN_JS)
+if [ $STATUS_CODE != "200" ]; then
+  echo "curl request to $ALLOY_MIN_JS returned status code $STATUS_CODE" 1>&2
+  exit 1
+fi
+
+ALLOY_JS="https://cdn1.adoberesources.net/alloy/${VERSION}/alloy.js"
+STATUS_CODE=$(curl -o /dev/null -Isw '%{http_code}\n' "${ALLOY_JS}")
```
Comment (Aaronius) [inline scripts/deploy.sh:53]: Is there an important reason why `"${ALLOY_JS}"` has quotes but `$ALLOY_MIN_JS` doesn't?

Looks good, though! 🚢 🇮🇹

**Comment by jfkhoury** [comment]
> @jfkhoury and @Aaronius I added a commit to verify that the CDN has the file. Do you see any problem with this?
> 
> [1244bfb](https://github.com/adobe/alloy/commit/1244bfbcac464a8ea0ec722ca1e9b31ec00de0bc)

Looks good! 🚢

**Pull request merged**